appium/webdriver/extensions/android/activities.py

```diff
@@ -12,15 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import TypeVar
+
 from selenium import webdriver
 from selenium.common.exceptions import TimeoutException
 from selenium.webdriver.support.ui import WebDriverWait
 
 from appium.webdriver.mobilecommand import MobileCommand as Command
 
+T = TypeVar('T', bound='Activities')
+
 
 class Activities(webdriver.Remote):
-    def start_activity(self, app_package, app_activity, **opts):
+    def start_activity(self, app_package: str, app_activity: str, **opts: str) -> T:
```

Comment:
`opts` can be passed without type

Reply:
same as https://github.com/appium/python-client/pull/482#discussion_r365553785